src/north/CsvToHttp/CsvToHttp.class.js

```diff
@@ -174,12 +267,50 @@ class CsvToHttp extends ApiHandler {
           object.error.push(`Header "${mapping.httpField}": ${response.error}`)
         }
         object.value[field] = response.value
+      } else if (mapping.csvField.match(REGEX_CONTAIN_VARIABLE_STRING)) {
+        // split the input into a array of string
+        // "test ${value}" => [ "test ", ${value}]
+        const csvFieldSplit = mapping.csvField.split(REGEX_SPLIT_TEMPLATE_STRING).filter(Boolean)
+        const field = mapping.httpField
+        csvFieldSplit.forEach((element) => {
+          // match if the string starts with: ${...}
+          if (element.match(REGEX_MATCH_VARIABLE_STRING)) {
+            const headerToGet = element.match(REGEX_GET_VARIABLE)
+            headerToGet.forEach((header) => {
```

Comment:
Correct me if I'm wrong, but from my understanding : headerToGet is the value extracted from element. If element is ${test}, so headerToGet will be [ 'test' ].

My point is : I don't see the case where you need to loop inside an array. It must always be one item inside the resulting array (because of the previous regex filtering). It may be clearer to remove this forEach. 
Instead, I would do something like this : 
const header element.match(REGEX_GET_VARIABLE)[0]
And then something like : if(header) { do stuff }

Reply:
If we have the input "${test} and ${test2}" the regex will match with both header. We'll get an array of both value. 
Here, we ensure that the headers "test" and "test2" are present in the CSV file. However, we notice the user that his input isn't valid.

Reply:
const csvFieldSplit = mapping.csvField.split(REGEX_SPLIT_TEMPLATE_STRING).filter(Boolean) should split "${test} and ${test2}" to ["${test}", " and ", "${test2}"] (line 273)

You then make a forEach on this array. So you parse individually each element. That's why I say that on line 278, you will always have only one item. So the forEach of line 279 is useless.

Am I right ?